packages/flutter_settings/lib/src/config/controls/dropdown.dart

```diff
@@ -0,0 +1,90 @@
+import "package:flutter/material.dart";
+import "package:flutter_settings/flutter_settings.dart";
+import "package:settings_repository/settings_repository.dart";
+
+/// Dropdown Control Configuration
+class DropdownControlConfig
+    extends DescriptiveTitleControlConfig<String, DropdownControlConfig> {
+  /// Constructor for Dropdown Control Config
+  DropdownControlConfig({
+    required super.title,
+    required super.description,
+    required super.initialValue,
+    required this.options,
+    this.hintText,
+    this.suffixIcon,
+    super.wrapperBuilder = defaultDescriptionTitleControlWrapper,
+  });
+
+  /// List of items for the dropdown control
+  final List<String> options;
+
+  /// Optional hint text displayed in the dropdown
+  final String? hintText;
+
+  /// Suffix icon for the dropdown
+  final Widget? suffixIcon;
+
+  @override
+  Widget buildSetting(
+    BuildContext context,
+    SettingsControl<String> control,
+    SettingsControlController controller,
+  ) =>
+      Expanded(
+        child: DropdownButtonFormField<String>(
+          icon: const Icon(
+            Icons.keyboard_arrow_down_rounded,
+          ),
+          items: options
+              .map(
+                (String item) => DropdownMenuItem<String>(
+                  value: item,
+                  child: Text(item),
+                ),
+              )
+              .toList(),
+          onChanged: (String? value) async {
+            if (value != null) {
+              await controller.updateControl(control.update(value));
+            }
+          },
+          value: control.value,
+          hint: Text(hintText ?? "Select an item"),
+          decoration: InputDecoration(
```

Comment:
please do not use hard styling on decoration like this.

Reply:
Done